adapters/ddf-adapter/src/main/java/com/connexta/replication/adapters/ddf/spring/ServiceConfig.java

```diff
@@ -28,14 +28,12 @@ public ClientFactoryFactory clientFactoryFactory() {
     return new ClientFactoryFactoryImpl();
   }
 
-  @Bean
-  public DdfRestClientFactory ddfRestClientFactory(ClientFactoryFactory clientFactoryFactory) {
-    return new DdfRestClientFactory(clientFactoryFactory);
-  }
-
   @Bean
   public DdfNodeAdapterFactory ddfNodeAdapterFactory(
```

Comment:
❗️ Javadoc missing

Reply:
I understand the class, but does the bean instantiation need javadoc?

Reply:
The method has parameters and those should be documented ... so yes